transactions/src/main/java/org/jboss/as/txn/subsystem/TransactionSubsystem31Parser.java

```diff
@@ -0,0 +1,209 @@
+/*
```

Comment:
OK, what does one do to make it an extension? Just extend 30Parser with the new code?